src/services/codefixes/inferFromUsage.ts

```diff
@@ -401,34 +414,35 @@ namespace ts.codefix {
 
         interface CallUsage {
             argumentTypes: Type[];
-            returnType: Usage;
+            return_: Usage;
         }
 
         interface Usage {
-            isNumber?: boolean;
-            isString?: boolean;
+            isNumber: boolean | undefined;

```

Comment:
cleanup: all Usage properties are now required and Usages are created with createEmptyUsage.